# Project suggestions


* RNA-seq analysis with cell type deconvolution: Follow the approach [here](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008120) and analyze an RNA-seq dataset generated at the FGCZ [taken]
* ~~Collaborative project (with Mark and a group from USZ): Analysis of single cell RNA-seq data: 1) to determine a suitable list of genes for in situ experiments; 2) integrating the local data with public datasets to find signatures of drug resistance (could actually be two projects)~~[taken]
* Methods project: understanding whether gene selection has an impact on performance for [SingleR](https://bioconductor.org/packages/release/bioc/html/SingleR.html) cell type prediction
* ~~Collaborative project (with Mark and a group from USZ): Analysis of single cell CITE-seq data: goals to be determined (roughly, do QC of the data, determine cell types, compare cell type compositions between normal, precursor, cancer cells)~~[taken]
* Methods project: compare methods to arrive at sets of marker genes (single cell RNA-seq)
* Methods project: normalization in differential abundance situations where the compositionality makes TMM challenging (e.g., cell type composition changes in single cell data)
* Analysis project: Impact of sequencing depth on single cell RNA-seq analysis
* Analysis project: Impact of doublets and doublet removal on single cell RNA-seq analysis
* Analysis project: Integration of different single-cell samples
